### ISSUE TITLE
validate hierarchy

### ISSUE THREAD

**Issue by Rdataflow** [zazuko/rdf-cube-schema#77]
hierarchies as described at [rdf-cube-schema/#hierarchies](https://zazuko.github.io/rdf-cube-schema/#hierarchies) shall be validated.

Feel free to profit from 
- some crafted data https://github.com/zazuko/rdf-cube-schema/commit/54a905bdbb0f61088f3937a3643d611304170238 (do not merge)
- [example implementation](https://github.com/zazuko/rdf-cube-schema/compare/main...Rdataflow:example-how-to-validate-inhierarchy?expand=1)

As always: review critically and feel free to come up with better implementation :-)
PS: depends on #75

**Comment by l00mi** [comment]
Please also try on: http://cube-viewer.zazuko.com/?cube=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104%2F8&endpointUrl=https%3A%2F%2Fint.ld.admin.ch%2Fquery&user=&password=&sourceGraph=

**Comment by l00mi** [comment]
And also: http://cube-viewer.zazuko.com/?cube=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104%2F8&endpointUrl=https%3A%2F%2Fint.ld.admin.ch%2Fquery&user=&password=&sourceGraph=

**Comment by Rdataflow** [comment]
> And also: http://cube-viewer.zazuko.com/?cube=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd0104%2F8&endpointUrl=https%3A%2F%2Fint.ld.admin.ch%2Fquery&user=&password=&sourceGraph=

This examples however do not feature the proposed `meta:inHierarchy` from the RDF cube schema spec. `meta:hasHierarchy` is not part of the spec or validation process. due to `sh:closed` `false` the validation is limited to known nodes and will pass...

**Comment by semanticfire** [comment]
Updates to shapes commited @Rdataflow please validate with your sample data using the shapes in the `documentation` branch
